common/djangoapps/contentserver/middleware.py

```diff
@@ -55,6 +56,19 @@ def process_request(self, request):
             except (ItemNotFoundError, NotFoundError):
                 return HttpResponseNotFound()
 
+            # Set the basics for this request.
+            newrelic.agent.add_custom_parameter('contentserver.course_id', loc.course_key)
+            newrelic.agent.add_custom_parameter('contentserver.org', loc.org)
```

Comment:
Sorry, I wasn't clear last time. Can we call this just `course_id` and `org`, instead of using `contentserver.course_id`/`contentserver.org`? That way, we can do queries like `select sum(duration) from Transaction where appName='ourapp' facet course_id` and get a split across different transaction types.